lib/openai/threads/runs.ex

```diff
@@ -0,0 +1,36 @@
+defmodule OpenAI.Threads.Runs do
+  @moduledoc false
+  alias OpenAI.Client
+  alias OpenAI.Config
+
+  @base_url "/v1/threads"
+
+  def url(thread_id), do: "#{@base_url}/#{thread_id}/runs"
+  def url(thread_id, run_id), do: "#{@base_url}/#{thread_id}/runs/#{run_id}"
```

Comment:
if we would like to have support for the function calling in the assistants. we would need to expose another endpoint `submit_tool_outputs` as described here: https://platform.openai.com/docs/assistants/tools/reading-the-functions-called-by-the-assistant